package.json

```diff
@@ -11,7 +11,7 @@
     "postpublish": "git push origin --follow-tags"
   },
   "devDependencies": {
-    "tap": "^14.10.7"
+    "tap": ">14.10.7"
```

Comment:
You never want to use > - the ^ already means “use the latest of v14.x”

Reply:
the issue is not fixed until 15.0.4. maybe I should use ^15.0.4